lib/internal/errors.js

```diff
@@ -386,6 +386,7 @@ class AssertionError extends Error {
     this.actual = actual;
     this.expected = expected;
     this.operator = operator;
+    this.rawMessage = lazyInternalUtil().removeColors(this.message);
```

Comment:
It seems like this could be a prototype getter, for performance?

Reply:
Hmmm... I'm not sure about this. Once we get a bit further along with assigning the error codes to the errors, I'd like to pursue adding these kinds of non-standard properties off a new `Error.prototype.info` bucket rather than directly off the `Error` object. "Why?", you may ask? The reason is to give a consistent common location for non-standard Additional Stuff.

Reply:
That is, to say, if this PR, for instance, added an `info` property with the `actual`, `expected`, `operator` and `rawMessage` properties to an `Error.prototype.info` and made the existing properties directly off the `Error` object into aliases that we could deprecate in 11.0.0, I'd be *much* happier with this.

Reply:
That (having access to the properties you mention) would actually be even better for our use case in Jest of transparently supporting `assert` but making it look native to Jest, so huge 👍 to that!

Reply:
@jasnell I guess the reason to have all additional properties on a single property is that it might conflict less with properties added else wise? I do not think that we should move all properties to a single property but we do lack documentation and more important out of my perspective: we should definitely export all our errors. That way the users could easily identify errors from Node.js core by just checking for being of the NodeError class. I think that is actually a better way to solve this problem than by moving everything to `info`.

Reply:
I changed it to a prototype getter and setter.